lib/query.js

```diff
@@ -5335,53 +5335,57 @@ Query.prototype.tailable = function(val, opts) {
  * @api private
  */
 
-Query.prototype.near = function() {
+function buildNearParams(args, sphere) {
```

Comment:
This isn't a very useful refactor, shuffling the logic to another function doesn't do anything.